recipes/pyupgrade/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyupgrade" %}
+{% set version = "1.11.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e4334e1400e619db06c45760e3b0e5c1ac6163e07a9778e530cde3304b516d9
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pyupgrade = pyupgrade:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - tokenize-rt >=2.1
+
+test:
+  commands:
+    - pyupgrade --help
+
+about:
+  home: https://github.com/asottile/pyupgrade
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
`{{ environ["RECIPE_DIR"] }}/LICENSE`

I'm not 100% sure but I believe you can do just `LICENSE` here, let conda find it on the recipe or the source, and be "future proof" when the license is available in the source and removed from the recipe.

Reply:
Yeah it worked for me locally, thanks, didn't know that trick yet. 👍